CMakeLists.txt

```diff
@@ -85,7 +84,10 @@ if(RANGE_V3_HEADER_CHECKS)
 else()
   add_custom_target(headers SOURCES ${RANGE_V3_PUBLIC_HEADERS_ABSOLUTE})
 endif()
-generate_standalone_header_tests(EXCLUDE_FROM_ALL MASTER_TARGET headers HEADERS ${RANGE_V3_PUBLIC_HEADERS})
```

Comment:
There's no reason now for `headers` to be attached to `all`; it doesn't build anything. We can remove lines 82-84 and 86 and just leave the fake target so IDEs know the header files are all part of the project.

Also, do you see any reason not to move the globs and include (lines 70 - 81) into the `if(RANGE_V3_TESTS)` on 87?

Reply:
The glob one for the IDEs was left outside without the `test/` prefix for its `FOLDER` property.

Reply:
> The glob one for the IDEs was left outside without the `test/` prefix for its `FOLDER` property.

Oh, obviously.